flake.nix

```diff
@@ -0,0 +1,15 @@
+{
+  description = "Development flake for changelog utils";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+  };
+
+  outputs = { self, nixpkgs }: {
+
+    packages.x86_64-linux.hello = nixpkgs.legacyPackages.x86_64-linux.hello;
+
+    packages.x86_64-linux.default = self.packages.x86_64-linux.hello;
+
+  };
```

Comment:
This needs to be adjusted to include the Rust stuff. Apparently it's not so super straight forward, some people are using some overlays to support Rust? E.g. check this: https://github.com/oxalica/rust-overlay